### PR TITLE
POLIO-1165: make lqas fields uneditable

### PR DESCRIPTION
**Pull request by quang-le**
Explain what problem this PR is resolving

Related JIRA tickets : POLIO-1165

## Self proofreading checklist

- [X] Did I use eslint and black formatters
- [X] Is my code clear enough and well documented
- [X] Are my typescript files well typed
- [ ] New translations have been added or updated if new strings have been introduced in the frontend
- [ ] My migrations file are included
- [ ] Are there enough tests
- [ ] Documentation has been included (for new feature)

## Changes

- Replace input fields with text fields

## How to test
Go to any campaign, check round tab

## Print screen / video

https://github.com/BLSQ/iaso/assets/38907762/9c8b88be-12b3-462b-9d77-a77c624a9b97

## Notes

Requires https://github.com/BLSQ/iaso/pull/779 and https://github.com/BLSQ/iaso-pipelines/pull/10

### CODE REVIEW

File: plugins/polio/js/src/forms/RoundForm.tsx [hunk 1]
```diff
@@ -70,6 +71,14 @@ export const RoundForm: FunctionComponent<Props> = ({ roundNumber }) => {
                     />
                 </Grid>
                 <Grid xs={12} md={6} item>
+                    <LqasDistrictsPassed
+                        lqasDistrictsFailing={
+                            rounds[roundIndex].lqas_district_failing
```
Comment (beygorghor) [inline plugins/polio/js/src/forms/RoundForm.tsx]: i would double check the presence of `rounds[roundIndex]` , it shouldn't happen but ...

File: plugins/polio/js/src/forms/RoundForm.tsx [hunk 2]
```diff
@@ -70,6 +71,14 @@ export const RoundForm: FunctionComponent<Props> = ({ roundNumber }) => {
...
+                        }
+                        lqasDistrictsPassing={
+                            rounds[roundIndex].lqas_district_passing
```
Comment (beygorghor) [inline plugins/polio/js/src/forms/RoundForm.tsx]: same here

File: plugins/polio/js/src/components/Rounds/LqasDistrictsPassed/LqasDistrictsPassed.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+                            )}: `}
+                        </Typography>
+                    </Grid>
+                    <Grid item xs={6} lg={8}>
+                        <Typography variant="button">
+                            {`${lqasDistrictsFailing ?? '--'}`}
+                        </Typography>
+                    </Grid>
+                </Grid>
+            </Box>
```
Comment (beygorghor) [inline plugins/polio/js/src/components/Rounds/LqasDistrictsPassed/LqasDistrictsPassed.tsx:51]: Those are a bit redundant  no ?

**Review by beygorghor — APPROVED** [review]
Nothing blocking for me

File: plugins/polio/js/src/components/Rounds/LqasDistrictsPassed/LqasDistrictsPassed.tsx (hunk 3 above)
Comment (quang-le) [reply plugins/polio/js/src/components/Rounds/LqasDistrictsPassed/LqasDistrictsPassed.tsx:51]: Yes, but I don't think the component is big enough to justify abstracting to a sub-component

**Pull request merged**